<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabosd</class>
 <widget class="QWidget" name="tabosd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="3" column="1" colspan="4">
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="horizontalSpacing">
      <number>30</number>
     </property>
     <property name="verticalSpacing">
      <number>-1</number>
     </property>
     <item row="13" column="2">
      <widget class="QLabel" name="slider_value_osd_margin_y">
       <property name="text">
        <string>22</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QSlider" name="horizontalSlider_osd_font_size">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>55</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QLabel" name="slider_value_osd_margin_x">
       <property name="text">
        <string>25</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QSlider" name="horizontalSlider_osd_margin_x">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>25</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QCheckBox" name="box_osd_margin_y">
       <property name="text">
        <string>osd-margin-y</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QCheckBox" name="box_osd_font_size">
       <property name="text">
        <string>osd-font-size</string>
       </property>
      </widget>
     </item>
     <item row="19" column="1">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="20" column="1">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_osd_position">
       <property name="text">
        <string>OSD position:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QCheckBox" name="box_osd_bar_w">
       <property name="text">
        <string>osd-bar-w</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QSlider" name="horizontalSlider_osd_blur">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QSlider" name="horizontalSlider_osd_border_size">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QLabel" name="slider_value_osd_bar_w">
       <property name="text">
        <string>75</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QCheckBox" name="box_osd_margin_x">
       <property name="text">
        <string>osd-margin-x</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QCheckBox" name="box_osd_border_size">
       <property name="text">
        <string>osd-border-size</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QSlider" name="horizontalSlider_osd_bar_align_y">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>-10</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSlider" name="horizontalSlider_osd_bar_align_x">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>-10</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QSlider" name="horizontalSlider_osd_bar_h">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="slider_value_osd_align_x">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QCheckBox" name="box_osd_bar_align_x">
       <property name="text">
        <string>osd-bar-align-x</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QSlider" name="horizontalSlider_osd_bar_w">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>75</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QLabel" name="slider_value_osd_bar_h">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QCheckBox" name="box_osd_bar_align_y">
       <property name="text">
        <string>osd-bar-align-y</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QSlider" name="horizontalSlider_osd_margin_y">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>22</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="15" column="2">
      <widget class="QLabel" name="slider_value_osd_font_size">
       <property name="text">
        <string>55</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QCheckBox" name="box_osd_bar_h">
       <property name="text">
        <string>osd-bar-h</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QLabel" name="slider_value_osd_align_y">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="16" column="2">
      <widget class="QLabel" name="slider_value_osd_border_size">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="17" column="2">
      <widget class="QLabel" name="slider_value_osd_shadow_offset">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QCheckBox" name="box_osd_shadow_offset">
       <property name="text">
        <string>osd-shadow-offset</string>
       </property>
      </widget>
     </item>
     <item row="14" column="2">
      <widget class="QLabel" name="slider_value_osd_blur">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QCheckBox" name="box_osd_blur">
       <property name="text">
        <string>osd-blur</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QSlider" name="horizontalSlider_osd_shadow_offset">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>40</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QCheckBox" name="box_osd_spacing">
       <property name="text">
        <string>osd-spacing</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QSlider" name="horizontalSlider_osd_spacing">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>40</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="18" column="2">
      <widget class="QLabel" name="slider_value_osd_spacing">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="21" column="1">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="spacing">
      <number>-1</number>
     </property>
     <item row="11" column="0">
      <widget class="QCheckBox" name="box_osd_scale_by_window">
       <property name="text">
        <string>osd-scale-by-window</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="box_osd_duration">
       <property name="text">
        <string>osd-duration</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QCheckBox" name="box_osd_playing_msg">
       <property name="text">
        <string>osd-playing-msg</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="box_osd_msg1">
       <property name="text">
        <string>osd-msg1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="box_no_osd">
       <property name="text">
        <string>no-osd-bar</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QCheckBox" name="box_osd_font_bold">
       <property name="text">
        <string>osd-bold</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="box_osd_msg2">
       <property name="text">
        <string>osd-msg2</string>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QComboBox" name="comboBox_osd_scale_by_window">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>yes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>no</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="valuebox_osd_msg3">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QComboBox" name="comboBox_osd_align_x">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>left</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>center</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>right</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="14" column="2">
      <widget class="QComboBox" name="comboBox_osd_bold">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>yes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>no</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLineEdit" name="valuebox_osd_status_msg">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="13" column="2">
      <widget class="QComboBox" name="comboBox_osd_align_y">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>top</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>center</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>bottom</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QLineEdit" name="valuebox_osd_playing_msg">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QComboBox" name="comboBox_osd_level">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QCheckBox" name="box_osd_align_y">
       <property name="text">
        <string>osd-align-y</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QCheckBox" name="box_osd_level">
       <property name="text">
        <string>osd-level</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QCheckBox" name="box_osd_status_msg">
       <property name="text">
        <string>osd-status-msg</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="box_osd_font">
       <property name="text">
        <string>osd-font</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="box_osd_msg3">
       <property name="text">
        <string>osd-msg3</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="valuebox_osd_duration">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QFontComboBox" name="fontComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="valuebox_osd_msg1">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QCheckBox" name="box_osd_align_x">
       <property name="text">
        <string>osd-align-x</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QCheckBox" name="box_osd_fractions">
       <property name="text">
        <string>osd-fractions</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="valuebox_osd_msg2">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_osd_general">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
       <property name="text">
        <string>General OSD Settings:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0,0">
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="spacing">
      <number>-1</number>
     </property>
     <item row="4" column="0">
      <widget class="QCheckBox" name="box_osd_border_color">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>osd-border-color</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="button_pick_osd_border_color">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>://images/colorpicker.png</normaloff>://images/colorpicker.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="box_osd_color">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>osd-color</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="button_pick_osd_back_color">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>://images/colorpicker.png</normaloff>://images/colorpicker.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="value_osd_color">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="button_pick_osd_color">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>://images/colorpicker.png</normaloff>://images/colorpicker.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="box_osd_back_color">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>osd-back-color</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="value_osd_border_color">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="value_osd_back_color">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="box_osd_shadow_color">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>osd-shadow-color</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="value_osd_shadow_color">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="button_pick_osd_shadow_color">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>://images/colorpicker.png</normaloff>://images/colorpicker.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_osd_coloring">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
       <property name="text">
        <string>OSD Color:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>box_osd_back_color</sender>
   <signal>toggled(bool)</signal>
   <receiver>button_pick_osd_back_color</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>764</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>765</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>box_osd_border_color</sender>
   <signal>toggled(bool)</signal>
   <receiver>button_pick_osd_border_color</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>844</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>845</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>box_osd_shadow_color</sender>
   <signal>toggled(bool)</signal>
   <receiver>button_pick_osd_shadow_color</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>884</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>885</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>box_osd_color</sender>
   <signal>toggled(bool)</signal>
   <receiver>button_pick_osd_color</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>804</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>805</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
